<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Balance</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DOW           DOI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SHARING the LOADING</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>139</y>
      <width>661</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="aircraft_type"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>A/C reg.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="aircraft_reg_num"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>seats</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="seats_complectation">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Qrew</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Aircraft_qrew">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>airport    Flight</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Flight"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>661</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>LOAD &amp; TRIM SHEET               WEIGHT KGs
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="change_coeff_bttn">
       <property name="text">
        <string>change coefficients</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="seatstableWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>461</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="cargotableWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>461</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>304</y>
      <width>160</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Cargo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="CargoSpinBox">
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>254</y>
      <width>160</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>PAX</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="PaxSpinBox">
       <property name="maximum">
        <number>200</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="checkbutton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QLabel" name="LastCheckStatus">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>460</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>160</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Dispatcher</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DispatcherName"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="DOW">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="DOI">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Manage seats</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Manage cargo</string>
    </property>
   </widget>
   <widget class="QLabel" name="errors">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>631</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="seats_ok">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>not checked</string>
    </property>
   </widget>
   <widget class="QLabel" name="cargo_ok">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>not checked</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>356</y>
      <width>160</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>FUEL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="FuelSpinBox">
       <property name="maximum">
        <number>16000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>160</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>TRIP FUEL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="TripFuelSpinBox">
       <property name="maximum">
        <number>16000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>160</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10"/>
   </widget>
   <widget class="QLabel" name="resoults">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>580</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="save_bttn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>save to file</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
